README.md

```diff
@@ -2,47 +2,93 @@
+* **Threat democratization**: AI agents will allow actors to emulate the scale of attacks previously seen with nation states. Going forward, the proverbial corner shop may need the same defenses as the pentagon. Target value needs to be reassessed.
+* **Accidental threats**: In the course of integrate AI for accelerating and improving software development and operations, AI models may leak secrets, open all ports on a firewall, or behave in an insecure manner as a result of improper training, tuning, or final configuration.
+* **Prompt injection attacks**: These attacks involve directly or indirectly injecting additional text into a prompt to influence the model’s output. As a result, it could lead to prompt leaking disclosing sensitive or confidential information.
+* **Membership inference attack**: Process determining if a specific data was part of the model’s training dataset. It is most relevant in the context of deep learning models and used to extract sensitive or private information included in the training dataset.
+* **Model vulnerability management**: Identifying techniques, mechanism, and practices to apply modern vulnerability managment identification, remediation, and management practices into the model use and model development ecosystem.
+* **Model integrity**: Developing mechanisms and tooling to provide secure software supply chain practices, assurances, provenance, and attestable metadata for models.
+
 Anyone is welcome to join our open discussions.
 
-WG Co-Chairs:
+## WG Leadership
+
+### Co-Chairs:
 
 Jay White - GitHub @camaleon2016
 
 Mihai Maruseac - GitHub @mihaimaruseac
 
-<!-- ## Objective -->
-
 <!-- ## Vision -->
 
-<!-- ## Scope -->
-
 ## How to Participate
 
 - We have bi-weekly meetings via Zoom. To join, please see the [OpenSSF Public Calendar](https://calendar.google.com/calendar/u/0/r?cid=czYzdm9lZmhwNWk5cGZsdGI1cTY3bmdwZXNAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ)
+  - [2024 Meeting Notes for the AIML WG](https://docs.google.com/document/d/1YNP-XJ9jpTjM6ekKOBgHH8-avAws2DVKeCpn858siiQ/edit)
 - Informal chat is welcome on the [OpenSSF Slack channel #wg_ai_ml_security](https://openssf.slack.com/archives/C0587E513KR) (these disappear over time)
 - Mailing list [openssf-wg-ai-ml-security](https://lists.openssf.org/g/openssf-wg-ai-ml-security)
 
 ## Current Work
 
-We welcome contributions, suggestions and updates to our projects. To contribute to work on GitHub, please fill in an issue or create a pull request..
+We welcome contributions, suggestions and updates to our projects. To contribute to work on GitHub, please fill in an issue or create a pull request.
```

Comment:
There is some work to clarify this, but I think we can merge this as it is and then clean up in the near future once clarification happens,.